src/components/storefront/ProductDetails/QuantityPicker.js

```diff
@@ -2,22 +2,44 @@ import * as React from 'react';
 import { Flex, IconButton, Input } from '@chakra-ui/react';
 import { AddIcon, MinusIcon } from '@chakra-ui/icons';
 
-const QuantityButton = ({ aria, icon, quantityProps }) => {
+const QuantityButton = ({ aria, icon, quantityProps, height, width }) => {
   return (
-    <IconButton size="icon" variant="inverted" borderRadius="none" aria-label={aria} icon={icon} {...quantityProps} />
+    <IconButton
+      height={height}
+      maxWidth={width}
+      size="icon"
+      variant="inverted"
+      borderRadius="none"
+      aria-label={aria}
+      icon={icon}
+      {...quantityProps}
+    />
   );
 };
 
-const QuantityPicker = ({ getInputProps, getIncrementButtonProps, getDecrementButtonProps }) => {
+const QuantityPicker = ({ getInputProps, getIncrementButtonProps, getDecrementButtonProps, size }) => {
   const inc = getIncrementButtonProps();
   const dec = getDecrementButtonProps();
   const input = getInputProps();
+  const dimension = size === 'checkout' ? '40px' : '52px';
```

Comment:
There probably is a better way to do this, but wasn't too sure where to do it

Reply:
I'll look into if there's a better way to do this tmrw, but this might be fine